java/vector/src/main/java/org/apache/arrow/vector/BitVector.java

```diff
@@ -344,6 +344,22 @@ public final void get(int index, NullableBitHolder holder) {
       holder.isSet = 1;
       holder.value = get(index);
     }
+
+    /**
+     * Get the number of bits set to 1
+     * @return the number of bits set to 1
+     */
+    @Override
+    public final int getNullCount() {
```

Comment:
This comment is incorrect. The null count is the number of bits set to 0 (up to the length of the vector)